metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -1613,10 +1611,6 @@ def _build_aggregated_measure_from_measure_source_node(
 
         # If querying an offset metric, join to time spine before aggregation.
         if before_aggregation_time_spine_join_description is not None:
-            assert queried_agg_time_dimension_specs, (
```

Comment:
Removed this because the `JoinToTimeSpineNode` will raise the same error in `__post_init__` on instantiation, so we don't need both.